src/test/java/com/wikia/webdriver/testcases/seotests/Our404HandlerTest.java

```diff
@@ -20,12 +20,11 @@
         {"pl.gta", "Specjalna:Version", "wiki/Specjalna:Wersja"},
         {"pl.gta", "Specjalna:Wersja", "wiki/Specjalna:Wersja"},
         {"www", "NoSuchPage", "NoSuchPage"},
-// Current code doesn't redirect to non-existing articles:
-//        {"wowwiki", "NoSuchPage", "wiki/NoSuchPage"},
-//        {"muppet", "NoSuchPage", "wiki/NoSuchPage"},
+        {"wowwiki", "NoSuchPage", "wiki/NoSuchPage"},
+        {"muppet", "NoSuchPage", "wiki/NoSuchPage"},
         {"muppet", "wiki/Honeysuckle_Rose", "wiki/Honeysuckle_Rose"},
         {"www", "About", "About"},
-// Does not work on Mercury:
+// Does not work on Mercury (XW-727):
 //        {"www", "wiki/About", "About"},
```

Comment:
Can you remove this please? this page is currently working on oasis and wikia mobile